plugins/admin/pkg/command/registry/add.go

```diff
@@ -42,7 +42,7 @@ func NewRegistryAddCommand(p *pkg.AdminParams) *cobra.Command {
 	var registryAddCmd = &cobra.Command{
 		Use:   "add",
 		Short: "Add registry with credentials",
-		Long:  `Add registry with credentials to enable Service deployment from it`,
+		Long:  `Add registry with credentials to enable service deployment from it`,
```

Comment:
This reads odd by ending with “it”. Perhaps better is:

“Add registry with credentials and enable service deployments from this registry” 

Or something similar.